Dockerfile

```diff
@@ -21,6 +21,7 @@ RUN ironic-inspector-dbsync --config-file /etc/ironic-inspector/inspector.conf u
 COPY ./runironic-inspector.sh /bin/runironic-inspector
 COPY ./runhealthcheck.sh /bin/runhealthcheck
 
+HEALTHCHECK CMD /bin/runhealthcheck
```

Comment:
I tested this using podman on the host Ironic but it doesn't seem to work - the healtcheck only runs once, not the default 30 seconds.  I assume this is why an external monitoring tool was added in OSP for podman -http://my1.fr/blog/openstack-containerization-with-podman-part-4-healthchecks/, since podman doesn't run as a daemon.

Reply:
Yes that's right, this doesn't work with podman, but you can do the external check.  With the command running in the foreground though the container will at least die if the process ends.  This healthcheck DOES work with kubernetes though (well once configured in the yaml).

Personally I think we could punt on the healthcheck on the host for now and figure that out later, as per the doc you linked.